docs/business.yaml

```diff
@@ -136,6 +136,84 @@ paths:
                         name: agmLocationChange
                         status: DRAFT
                         submitter: api-key-account-3445-sandbox
+                consent-continuation-out-success-response:
+                  summary: Consent Continuation Out Response
+                  value:
+                    filing:
+                      business:
+                        legalName: 0882848 B.C. LTD.
+                        legalType: BEN
+                        identifier: BC0882848
+                        foundingDate: '2024-07-08T15:34:57.844764+00:00'
+                      consentContinuationOut:
+                        foreignJurisdiction:
+                          country: CA
+                          region: SK
+                        courtOrder:
+                          fileNumber: '12345'
+                          effectOfOrder: planOfArrangement
+                      header:
+                        affectedFilings: []
+                        availableOnPaperOnly: false
+                        certifiedBy: "string"
+                        colinIds: []
+                        comments: []
+                        date: '2024-07-18'
+                        deletionLocked: false
+                        documentOptionalEmail: "user@example.com"
+                        effectiveDate: '2024-07-18T07:00:42.552434+00:00'
+                        email: "user@example.com"
+                        filingId: 150056
+                        inColinOnly: false
+                        isCorrected: false
+                        isCorrectionPending: false
+                        isPaymentActionRequired: true
+                        name: consentContinuationOut
+                        paymentAccount: '3445'
+                        paymentStatusCode: 'CREATED'
+                        paymentToken: '38416'
+                        status: 'PENDING'
+                        submitter: 'api-key-account-3445-sandbox'
```

Comment:
Please update to use "mocked submitter". 

As per our conversation, Eason will update any other references to `submitter: 'api-key-account-3445-sandbox'` in his upcoming work.

Reply:
Updated